packages/lesswrong/components/languageModels/LanguageModelChat.tsx

```diff
@@ -414,50 +655,39 @@ export const ChatInterface = ({classes}: {
     archiveConversation(conversationId);
   };
 
-  const conversationSelect = <Select 
-    onChange={onSelect} 
-    value={currentConversation?._id ?? NEW_CONVERSATION_MENU_ITEM}
+  const ragModeSelect = <Select 
```

Comment:
This doesn't look like a change to the RAG Mode select, but does look like it's getting rid of the conversation select entirely?